dsew_community_profile/delphi_dsew_community_profile/pull.py

```diff
@@ -597,9 +626,29 @@ def unify_testing_sigs(positivity_df, volume_df):
     https://docs.google.com/document/d/1MoIimdM_8OwG4SygoeQ9QEVZzIuDl339_a0xoYa6vuA/edit#
 
     """
-    # Combine test positivity and test volume, maintaining "this week" and "previous week" status.
+    pos_groups_df = positivity_df.drop(
+        ["val", "se", "sample_size", "timestamp"],
+        axis=1
+    ).drop_duplicates(
+    )
+    vol_groups_df = volume_df.drop(
+        ["val", "se", "sample_size", "timestamp"],
+        axis=1
+    ).drop_duplicates(
+    )
+    merged = pos_groups_df.merge(
+        vol_groups_df,
+        on=["geo_id", "publish_date"],
+        how="outer",
+        indicator=True
+    )
+    assert all(merged["_merge"] == "both"), \
+        "Each publish date-geo value combination should be available for both " + \
+        "test positivity and test volume."
```

Comment:
Community_Profile_Report_20220225.xlsx has 158 counties where positivity is not available for "last week" but volume is, and 7 counties where volume is not available but positivity is. Will this assert inappropriately reject that file? Or do you mean that previous steps should have already dropped the entries with mismatched availability?

Reply:
The wording may be ambiguous on my part. "available" here means that there is a row for a given region and date. It doesn't check if the signal is `NA`or not. `pandas.read_excel` reads empty fields as `NA/NaN`, so obs are included as `NA` where the signal was not available in the original report.

The way the sample size filter is set up at the moment, any obs with missing (`NA`) n or n <= 5 are dropped.

Reply:
Rows are included if `val` (test positivity) is missing but sample size is available and > 5.